lib/connection/connection.js

```diff
@@ -9,7 +9,8 @@ var inherits = require('util').inherits
   , debugOptions = require('./utils').debugOptions
   , Response = require('./commands').Response
   , MongoError = require('../error')
-  , Logger = require('./logger');
+  , Logger = require('./logger')
+  , Snappy = require('snappy');
```

Comment:
Snappy must be referenced like bson-ext using require_optional as we cannot rely on the user having or wanting to instal the native lib.